doc/src/manual/environment-variables.md

```diff
@@ -136,182 +131,183 @@ Suppose the value of `$JULIA_PKGRESOLVE_ACCURACY` is `n`. Then
+    garbage-collection debugging (that is, if `WITH_GC_DEBUG_ENV` is set to `1` in the build
+    configuration).
 
 ### `JULIA_GC_NO_GENERATIONAL`
 
-If set to anything besides `0`, then the Julia garbage collector never performs
-"quick sweeps" of memory.
+If set to anything besides `0`, then the Julia garbage collector never performs "quick
+sweeps" of memory.
 
 !!! note
 
-    this environment variable only has an effect if Julia was compiled
-    with garbage-collection debugging (that is, if `WITH_GC_DEBUG_ENV` is set
-    to `1` in the build configuration).
+    This environment variable only has an effect if Julia was compiled with
+    garbage-collection debugging (that is, if `WITH_GC_DEBUG_ENV` is set to `1` in the build
+    configuration).
 
 ### `JULIA_GC_WAIT_FOR_DEBUGGER`
 
-If set to anything besides `0`, then the Julia garbage collector will wait for
-the debugger to attach instead of aborting whenever there's a critical error.
+If set to anything besides `0`, then the Julia garbage collector will wait for the debugger
+to attach instead of aborting whenever there's a critical error.
 
 !!! note
 
-    this environment variable only has an effect if Julia was compiled
-    with garbage-collection debugging (that is, if `WITH_GC_DEBUG_ENV` is set
-    to `1` in the build configuration).
+    This environment variable only has an effect if Julia was compiled with
+    garbage-collection debugging (that is, if `WITH_GC_DEBUG_ENV` is set to `1` in the build
+    configuration).
 
 ### `ENABLE_JITPROFILING`
 
-If set to anything besides `0`, then the compiler will create and register an
-event listener for just-in-time (JIT) profiling.
+If set to anything besides `0`, then the compiler will create and register an event listener
+for just-in-time (JIT) profiling.
 
 !!! note
 
-    this environment variable only has an effect if Julia was compiled
-    with JIT profiling support, using either
+    This environment variable only has an effect if Julia was compiled with JIT profiling
+    support, using either
 
-*   Intel's [VTune™
-    Amplifier](https://software.intel.com/en-us/intel-vtune-amplifier-xe)
+*   Intel's [VTune™ Amplifier](https://software.intel.com/en-us/intel-vtune-amplifier-xe)
     (`USE_INTEL_JITEVENTS` set to `1` in the build configuration), or
-*   [OProfile](http://oprofile.sourceforge.net/news/) (`USE_OPROFILE_JITEVENTS`
-    set to `1` in the build configuration).
+*   [OProfile](http://oprofile.sourceforge.net/news/) (`USE_OPROFILE_JITEVENTS` set to `1`
+    in the build configuration).
 
 ### `JULIA_LLVM_ARGS`
 
 Arguments to be passed to the LLVM backend.
 
 !!! note
 
-    this environment variable has an effect only if Julia was compiled with
-    `JL_DEBUG_BUILD` set.
+    This environment variable has an effect only if Julia was compiled with `JL_DEBUG_BUILD`
+    set—in particular, the `julia-debug` executable is always compiled with this build
```

Comment:
I'd put spaces around the dash, otherwise it looks like "set-in" is being used as a single hyphenated word